<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogstock</class>
 <widget class="QDialog" name="dialogstock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_music">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Musique</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Id produit</string>
     </property>
    </widget>
    <widget class="QLabel" name="date">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>380</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Id stock</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="reservation">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>380</y>
       <width>791</width>
       <height>501</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id produit</string>
       </property>
      </widget>
      <widget class="QLabel" name="date_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>380</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Id stock</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="stock_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>561</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="reservation_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>561</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_14">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tabstock_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>781</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="tridescid_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>380</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Changer Tri Desc (ID)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tridescres_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>380</y>
         <width>261</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Changer Tri Desc(Date Res)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="triascid_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>380</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Changer Tri Asc (ID)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="triascres_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>380</y>
         <width>261</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Changer Tri Asc(Date Res)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="triascdate_2">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>380</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Changer Tri Asc(Date)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tridesdate_2">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>380</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Changer Tri Desc(Date)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rech_4">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>1</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_15">
      <attribute name="title">
       <string>Supprimer</string>
      </attribute>
      <widget class="QLineEdit" name="stocksup_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>581</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>26</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID STOCK</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer_4">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_16">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QPushButton" name="modfierOk_2">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>240</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Sauvgarder</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="stockEdit_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_2">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>60</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Stock :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="reservationEdit_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>250</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nouveau Date:</string>
       </property>
      </widget>
      <widget class="QLabel" name="date2_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>186</y>
         <width>55</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>350</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>STOCK</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="button_music_2">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Musique</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>910</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Return to Main Menu</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QTableView" name="tabstock">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>781</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="tridescid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Desc (ID)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tridescres">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>380</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Desc(Date Res)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triascid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Asc (ID)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triascres">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>380</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Asc(Date Res)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triascdate">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>380</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Asc(Date)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tridesdate">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>380</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Desc(Date)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rech_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>1</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_11">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLineEdit" name="stocksup">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>581</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>26</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID STOCK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_12">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QPushButton" name="modfierOk">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>240</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sauvgarder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stockEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>60</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Stock :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="reservationEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>250</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau Date:</string>
     </property>
    </widget>
    <widget class="QLabel" name="date2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>186</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>590</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Main Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>STOCK</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
